package.json

```diff
@@ -19,9 +19,9 @@
     "babel-plugin-transform-es2015-template-literals": "^6.5.0",
     "babel-plugin-transform-es3-member-expression-literals": "^6.5.0",
     "babel-plugin-transform-es3-property-literals": "^6.5.0",
-    "babel-plugin-transform-jscript": "^6.5.0",
     "babel-plugin-transform-exponentiation-operator": "^6.5.0",
-    "babel-preset-es2015": "^6.5.0",
+    "babel-plugin-transform-jscript": "^6.5.0",
+    "babel-preset-es2015-without-strict": "0.0.2",
```

Comment:
I don't think that will make any difference.

![screen shot 2016-05-06 at 11 45 23 am](https://cloud.githubusercontent.com/assets/195534/15083066/14585b76-1380-11e6-9f55-1b9c4d885ed3.png)